src/host/main.cpp

```diff
@@ -117,10 +117,20 @@ int main(int argc, char** argv)
     "Address to advertise publicly to clients (defaults to same as "
     "--rpc-address)");
 
-  std::string ledger_file("ccf.ledger");
-  app.add_option("--ledger-file", ledger_file, "Ledger file")
+  std::string ledger_dir("ledger");
+  app.add_option("--ledger-dir", ledger_dir, "Ledger directory")
     ->capture_default_str();
 
+  size_t ledger_chunk_threshold = 23; // ~8MB
```

Comment:
CLI11 has a very nice facility for this:

```
CLI::AsSizeValue(...): Convert inputs like 100b, 42 KB, 101 Mb, 11 Mib to absolute values. KB can be configured to be interpreted as 10^3 or 2^10.
```

I think it would be much more user friendly that having to specify a power of two (even if we do end up rounding up to that for some reason).

Reply:
I didn't know about this - that's excellent! We are now using it for the ledger chunk size.